multicluster/charts/linkerd-multicluster/Chart.yaml

```diff
@@ -11,7 +11,7 @@ kubeVersion: ">=1.21.0-0"
 name: "linkerd-multicluster"
 sources:
 - https://github.com/linkerd/linkerd2/
-version: 30.3.1-edge
+version: 30.3.2-edge
```

Comment:
Since the chart was updated with new values, I wonder if we should bump it to `30.4.0` instead. I'm hazy no these things myself, cc @alpeb

Reply:
We used to bump the minor segment for values.yml changes and the patch for anything else, but since we changed things so stable and edge follow the same (although intertwined) progression, the minor is now bumped when an edge comes right after a stable, to leave room for stable upgrades in that same branch.
This is just to say that a minor bump doesn't always mean a biggish change (like values.yaml changes). But it doesn't hurt if we do it in this case.